cool-core/src/main/java/com/nus/cool/core/schema/FieldType.java

```diff
@@ -71,4 +71,19 @@ public static FieldType fromInteger(int i) {
         throw new IllegalArgumentException("Invalid field type int: " + i);
     }
   }
+
+  public static boolean IsHashType(FieldType fieldType) {
```

Comment:
Personally,  I think this method is necessary since it integrate the logic to determine whether it use set type or range type.
Why it is only used in CohortProcessor ? Because old version code use swicth to do it.
Check code in `DataChunkWS` line 103.
Check code in `MetaChunkWS` line line 95.